easy-coding-standard.yml

```diff
@@ -0,0 +1,191 @@
+    PhpCsFixer\Fixer\ArrayNotation\TrailingCommaInMultilineArrayFixer: ~
+    PhpCsFixer\Fixer\ArrayNotation\TrimArraySpacesFixer: ~
+    PhpCsFixer\Fixer\Operator\UnaryOperatorSpacesFixer: ~
+    PhpCsFixer\Fixer\ClassNotation\VisibilityRequiredFixer: ~
+    PhpCsFixer\Fixer\ArrayNotation\WhitespaceAfterCommaInArrayFixer: ~
+    PhpCsFixer\Fixer\ControlStructure\YodaStyleFixer:
+        equal: false
+        identical: false
+        less_and_greater: false
+
+    # Class and Interface names should be unique in a project, they should never be duplicated
+    PHP_CodeSniffer\Standards\Generic\Sniffs\Classes\DuplicateClassNameSniff:
+    # Incrementers in nested loops should use different variable names
+    PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\JumbledIncrementerSniff:
+    # Control Structures must have at least one statement inside of the body (empty catch rules is skipped)
+    PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\EmptyStatementSniff:
+    # Line length should not exceed 120 characters
+    PHP_CodeSniffer\Standards\Generic\Sniffs\Files\LineLengthSniff:
+        lineLimit: 120
+        absoluteLineLimit: 0
+    # Empty catch statements must have comment explaining why the exception is not handled
+    PHP_CodeSniffer\Standards\Squiz\Sniffs\Commenting\EmptyCatchCommentSniff:
+    # For loops that have only a second expression (the condition) should be converted to while loops
+    PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\ForLoopShouldBeWhileLoopSniff:
+    # If statements that are always evaluated should not be used
+    PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\UnconditionalIfStatementSniff:
+    # Methods should not be declared final inside of classes that are declared final
+    PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\UnnecessaryFinalModifierSniff:
+    # Methods should not be defined that only call the parent method
+    PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\UselessOverridingMethodSniff:
+    # There should only be one class defined in a file
+    PHP_CodeSniffer\Standards\Generic\Sniffs\Files\OneClassPerFileSniff:
+    # There should only be one interface defined in a file
+    PHP_CodeSniffer\Standards\Generic\Sniffs\Files\OneInterfacePerFileSniff:
+    # There should only be one trait defined in a file
+    PHP_CodeSniffer\Standards\Generic\Sniffs\Files\OneTraitPerFileSniff:
+    # Exactly one space is allowed after a cast
+    PHP_CodeSniffer\Standards\Generic\Sniffs\Formatting\SpaceAfterCastSniff:
+    # Some functions should not appear in the code
+    PHP_CodeSniffer\Standards\Generic\Sniffs\PHP\ForbiddenFunctionsSniff:
+        forbiddenFunctions:
+            var_dump: null
+            dump: null
+            echo: null
+            require: null
+            require_once: null
+            include: null
+            include_once: null
+            phpinfo: null
+            eval: null
+            sizeof: count
+            printf: null
```

Comment:
what about `join: implode`?

Reply:
Its already part of [NoAliasFunctionsFixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/f9d77889231a3495f100cca1857403d80506d6e7/src/Fixer/Alias/NoAliasFunctionsFixer.php#L29-L55). Also this means sizeof is redundant as well, I will remove it.